frontend/apps/battleship-zk/package.json

```diff
@@ -4,9 +4,9 @@
   "version": "1.0.2",
   "type": "module",
   "scripts": {
-    "start": "yarn build:packages && yarn build:ez-transactions && vite --open",
-    "build": "yarn build:packages && yarn build:ez-transactions && tsc && vite build",
-    "preview": "vite preview"
+    "start": "yarn build:packages && yarn build:ez-transactions && npx vite --open",
```

Comment:
Why use `npx` instead of `yarn vite --open`?

Reply:
Since all `devDependencies` are in root `package.json` right now, `npx` helps to resolve `vite` to root `node_modules` directory.